apisix/balancer/ewma.lua

```diff
@@ -1,3 +1,20 @@
+--
```

Comment:
This is wrong. This file's LICENSE has been specified in the LICENSE separately.